src/Model/CollectionInfo.php

```diff
@@ -59,6 +59,8 @@ public function __debugInfo()
     /**
      * Return the maximum number of documents to keep in the capped collection.
      *
+     * @deprecated Deprecated in favor of using getOptions
```

Comment:
For consistency with the other two places we use `@deprecated` (ChangeStream and Collection), I think this should include the version in which we deprecate it. Consider:

```
@deprecated 1.9 Deprecated in favor of using getOptions
```